test/connectSpec.js

```diff
@@ -30,12 +30,18 @@ describe('Connection methods', function () {
 
     this.wsRedLedger = wsHelper.makeServer('ws://red.example/websocket?token=abc')
     this.infoRedLedger = cloneDeep(require('./data/infoRedLedger.json'))
+
+    this.accountNock = nock('http://red.example')
```

Comment:
If this nock is only needed for one test, you should just include it in that test